src/content/en/2021/javascript.md

```diff
@@ -1,20 +1,1543 @@
+
+This is where the recent advances in the HTTP protocol come in and the idea of reducing the number of JavaScript requests for better performance gets inaccurate.  With the introduction of HTTP/2 and HTTP/3 the overhead of HTTP requests has significantly reduced so requesting the same resources over more requests is not necessarily a bad thing anymore.
+
+Read more about the state of the protocols in the [HTTP](./http) chapter.
+
+
+### module and nomodule
+
+When loading a website, the browser renders the HTML and requests the appropriate resources. It consumes the polyfills referenced in the code for the effective rendering and functioning of the page. The modern browsers that support newer syntax like [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), [async functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function), etc. do not need loads of polyfills to make things work and therefore, should not have to.
+
+This is when differential loading takes care of things. Specifying the `type=”module”` attribute would serve the modern browsers the bundle with modern syntax and fewer polyfills, if any. Similarly, older browsers that lack support for modules will be served the bundle with required polyfills and transpiled code syntax with attribute `type=”nomodule”`. Read more about the usage of modue/nomodule [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#applying_the_module_to_your_html)
+
+Let's look at the data to understand the adoption of these attributes. \
+4.6% of desktop pages use the attribute `type=”module”` whereas only 3.9% of mobile pages use `type=”nomodule”`. This could be due to the fact that the mobile dataset being much larger contains more "long-tail" websites that might not be using the latest features.
+
+<figure markdown>
+<table>
+  <tr>
+   <td><strong>client</strong>
+   </td>
+   <td><strong>module</strong>
+   </td>
+   <td><strong>nomodule</strong> 
+   </td>
+  </tr>
+  <tr>
+   <td><em>Desktop</em>
+   </td>
+   <td>4.6%
+   </td>
+   <td>3.9%
+   </td>
+  </tr>
+  <tr>
+   <td><em>Mobile</em>
+   </td>
+   <td>4.3%
+   </td>
+   <td>3.9%
+   </td>
+  </tr>
+</table>
+
+<figcaption>{{ figure_link(caption="Distribution of differential loading usage on desktop and mobile clients.", sheets_gid="1261294750", sql_file="module_and_nomodule.sql") }}</figcaption>
+</figure>
+
+It is important to note that with the [end of support for IE11 browser](https://docs.microsoft.com/en-us/lifecycle/announcements/internet-explorer-11-support-end-dates), differential loading is now well supported across all browsers that most sites support.
```

Comment:
```suggestion
It is important to note that with the [end of support for IE11 browser](https://docs.microsoft.com/en-us/lifecycle/announcements/internet-explorer-11-support-end-dates), differential loading is less applicable because evergreen browsers support modern JavaScript syntax.
```

Reply:
You can support this statement with https://github.com/angular/angular/issues/41840.